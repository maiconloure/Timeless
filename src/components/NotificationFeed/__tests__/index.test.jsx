import React from 'react';
import renderer from 'react-test-renderer';

import FeedCard from '../index';

describe('Snapshot', () => {
  it('renders component', () => {
    const tree = renderer
      .create(<FeedCard prop={['Está muito parado por aqui... crie novos cartões!']} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
