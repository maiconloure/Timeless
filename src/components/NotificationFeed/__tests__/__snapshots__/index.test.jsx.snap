// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot renders component 1`] = `
<div
  className="sc-hiKfjK kmjzEC"
  height="450px"
  width="280px"
>
  <div
    className="sc-gXfWUo cTZZDr"
  >
    <div
      className="sc-cBoprd hXbwCO"
    >
      <h2>
        Notificações recentes
      </h2>
    </div>
    <div
      className="sc-ciSmjq jLvfax"
      onClick={[Function]}
    >
       - 
    </div>
  </div>
  <div
    className="sc-jcwofb sc-carGAA kNhOZX gXYsNS"
  >
    <div
      className="sc-iTVIwl jiAQaA"
    >
      <p
        className="sc-iBzFoy igVgod"
        color="#014D82"
        fontSize="1.5rem"
      >
        Está muito parado por aqui... crie novos cartões!
      </p>
    </div>
  </div>
</div>
`;
